code/controllers/subsystem/shuttles.dm

```diff
@@ -254,7 +254,7 @@ var/datum/subsystem/shuttle/SSshuttle
 	slip.ordernumber = ordernum
 
 	var/stationName = (errors & MANIFEST_ERROR_NAME) ? new_station_name() : station_name()
-	var/packagesAmt = SSshuttle.shoppinglist.len + ((errors & MANIFEST_ERROR_COUNT) ? 0 : rand(1,2))
```

Comment:
MANIFEST_ERROR_COUNT flag should be set when we want there to be an error in the count.
The same way MANIFEST_ERROR_NAME flag being set indicates to use an incorrect station name.